CHANGELOG.adoc

```diff
@@ -21,6 +21,7 @@
 === Bug fixes
 
 - https://github.com/eclipse-sirius/sirius-components/issues/1301[#1301] [core] Add support for having intermodel references
+- https://github.com/eclipse-sirius/sirius-components/issues/1301[#1392] [core] Fix EditingContextCrossReferenceAdapter for derived references
```

Comment:
- Invalid issue referenced
- The previous pull request on that topic indicated that the only breaking change was a simple rename of `SiriusWebJSONResourceFactoryImpl`. Could you provide more precise information since the change on the URI was quite significant and will impact many consumers of Sirius Consumers
- The previous pull request introduced an update in the version of EMFJson and it was described in the wrong section of the ADR, could you fix that here

Reply:
Not sure to understand you last sentence : " it was described in the wrong section of the ADR"

I moved this part in breaking changes section
```
- https://github.com/eclipse-sirius/sirius-components/issues/1301[#1301] [core] Add support for having intermodel references. Now, the URI of resources added in a ResourceSet must have a `scheme`. Otherwise, you will  encounter exceptions. This version is based on `org.eclipse.sirius.emfjson` version 2.3.2.
```

Reply:
Your entry about the bump to EMFJson 2.3.2 is in the release notes of 2022.9.0 instead of 2022.11.0. @pcdavid told you to change it in the comment of the previous pull request.

Reply:
I also changed the `Dependency update` section